Constitution.txt

```diff
@@ -1,490 +1,264 @@
 
 Section 9
-1: The Migration or Importation of such Persons as any of the States now existing shall think proper to admit, shall not be prohibited by the Congress prior to the Year one thousand eight hundred and eight, but a Tax or duty may be imposed on such Importation, not exceeding ten dollars for each Person.
+1: The Privilege of the Writ of Habeas Corpus shall not be suspended, unless when in Cases of Rebellion or Invasion the public Safety may require it, whereupon it shall be suspended for more than one week only with the approval of three-fourths of both the House of Representatives and the Senate.
 
-2: The Privilege of the Writ of Habeas Corpus shall not be suspended, unless when in Cases of Rebellion or Invasion the public Safety may require it.
+2: No Bill of Attainder or ex post facto Law shall be passed.
 
-3: No Bill of Attainder or ex post facto Law shall be passed.
+3: No Capitation, or other direct, Tax shall be laid.
 
-4: No Capitation, or other direct, Tax shall be laid, unless in Proportion to the Census or Enumeration herein before directed to be taken.7
+4: No Tax or Duty shall be laid on Articles exported from any State, unless such Tax or Duty shall be imposed uniformly upon all States.
 
-5: No Tax or Duty shall be laid on Articles exported from any State.
+5: No Preference shall be given by any Regulation of Commerce or Revenue to the Ports of one State over those of another: nor shall Vessels bound to, or from, one State, be obliged to enter, clear, or pay Duties in another.
 
-6: No Preference shall be given by any Regulation of Commerce or Revenue to the Ports of one State over those of another: nor shall Vessels bound to, or from, one State, be obliged to enter, clear, or pay Duties in another.
+6: No Money shall be drawn from the Treasury, but in Consequence of Appropriations made by Law; and a regular Statement and Account of the Receipts and Expenditures of all public Money shall be published at least once per year. Such Statement shall include a record of any debts or surpluses arising from any individual Bill.
 
-7: No Money shall be drawn from the Treasury, but in Consequence of Appropriations made by Law; and a regular Statement and Account of the Receipts and Expenditures of all public Money shall be published from time to time.
+7: No Title of Nobility shall be granted by these United States: And no Person holding any Office of Profit or Trust under them, shall, without the Consent of the Congress, accept of any present, Emolument, Office, or Title, of any kind whatever, from any King, Prince, or foreign State. No person holding a Title of Nobility in a State shall have any authority relating to his Title in any other State, nor shall he have any authority over the Union.
```

Comment:
I imply here that a State may have titles of nobility upon its entrance into the Union, and may even establish internal titles of nobility. This is a specific outreach to nations which have formal royalty, but operate as a constitutional republic. If the United Kingdom were to establish a written constitution, formalizing its current parliamentary government, it could (if they desire) retain the Royal Family as a formal element of the government, and recognize Queen Elizabeth II as the formal Chief Executive. They simply would need to recognize that she holds no authority over any other Citizen outside of the UK.